src/Features/Core/Portable/UnifiedSuggestions/UnifiedSuggestedActionsSource.cs

```diff
@@ -342,7 +342,7 @@ static CodeFixGroupKey GetGroupKey(CodeFix fix)
                     }
                 }
 
-                var combinedSpan = minStart >= 0 ? new TextSpan(minStart, maxEnd) : (TextSpan?)null;
+                var combinedSpan = minStart >= 0 ? new TextSpan(minStart, maxEnd - minStart) : (TextSpan?)null;
```

Comment:
This was a bug in the existing code that used the end point instead of the length.